modules/core/src/transitions/transition-interpolator.js

```diff
@@ -32,7 +32,11 @@ export default class TransitionInterpolator {
    */
   arePropsEqual(currentProps, nextProps) {
     for (const key of this._propsToCompare || Object.keys(nextProps)) {
-      if (!equals(currentProps[key], nextProps[key])) {
+      if (
```

Comment:
Why is this changing?

Reply:
@ibgreen `equals` in 2.3 returns `false` when comparing `undefined, undefined`

Reply:
I see. Comparing `undefined`s seems like an edge case that we could potentially define as we choose, though I can't see a good reason to not have them compare as equal, so I assume we are fine with changing the code here.